docs/site/DataSources.md

```diff
@@ -81,3 +81,17 @@ Attach the newly created datasource to the app by calling `app.dataSource()`.
 The `app.datasource()` method is available only on application classes
 with `RepositoryMixin` applied.
 " %}
+
+### Connector
+
+<!-- TODO
```

Comment:
Please add a short note to explain readers that this is a placeholder page and add a link to the GH issue that's tracking the task of writing the actual content.